incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,163 @@
+## Pod Annotations
+# podAnnotations: {}
+
+## Statefulset annotations
+# annotations: {}
+
+## Number of ClickHouse nodes
+replicas: 1
+
+## Image settings
+## ref: http://kubernetes.io/docs/user-guide/images/
+image: yandex/clickhouse-server
+imageTag: latest
+imagePullPolicy: Always
```

Comment:
This should be `IfNotPresent` to go along with the tag. No need to constantly download.

Reply:
Agree, fixed